dc/core/resources/core.qrc

```diff
@@ -46,5 +46,13 @@
         <file>FocusControlButton.qml</file>
         <file>ControlPanel.qml</file>
         <file>ControlPanelDelegate.qml</file>
+        <file>WallDisplayGroup.qml</file>
+    </qresource>
+    <qresource prefix="/clock">
+        <file>Clock.qml</file>
+        <file>clock.svg</file>
```

Comment:
Well, qml files have to use upper case because they define a "class" of the same name (http://doc.qt.io/qt-4.8/qml-best-practices-coding.html). For images we can do what we want, but we typically don't use upper case... That's the convention that I used for the previous resources at least.